src/OpenTelemetry.Instrumentation.AspNet/README.md

```diff
@@ -137,6 +137,18 @@ below.
 
 ### Trace Filter
 
+> [!NOTE]
+> OpenTelemetry has the concept of a
+[Sampler](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/sdk.md#sampling).
+When using `OpenTelemetry.Instrumentation.AspNet.TelemetryHttpModule` the
+`url.path` tag is supplied automatically to samplers when telemetry is started
+for incoming requests. It is recommended to use a sampler which inspects
+`url.path` (as opposed to the `Filter` option described below) in order to
+perform filtering as it will prevent child spans from being created and bypass
```

Comment:
```suggestion
perform filtering as it can ensure consistent sampling when ParentBased samplers are used, by consistently recording/not-recording this Activity and the children..
```

Not sure of how best to phrase this. Will come back later. This is reasonably good to merge now.